Index: src/main/java/com/eshop/TenWeb/Model/application.java
===================================================================
diff --git a/src/main/java/com/eshop/TenWeb/Model/application.java b/src/main/java/com/eshop/TenWeb/Model/application.java
new file mode 100644
--- /dev/null	(date 1726477284768)
+++ b/src/main/java/com/eshop/TenWeb/Model/application.java	(date 1726477284768)
@@ -0,0 +1,6 @@
+package com.eshop.TenWeb.Model;
+
+public class application {
+    public static void main(String[] args) {
+    }
+}
